traffic_ops/traffic_ops_golang/server/servers_assignment.go

```diff
@@ -390,12 +389,6 @@ INSERT INTO deliveryservice_server (deliveryservice, server)
 			} else {
 				delete = append(delete, param)
 			}
-			param = getConfigFile(cacheURLPrefix, xmlID.String)
```

Comment:
We can't get rid of this functionality yet, right? As long as the 3.1 and below APIs exist, the field still exists, and its associated functionality should be supported?

Reply:
Correct, I've readdded it